### DISCUSSION TITLE
Is there a halt on community PRs?

### DISCUSSION THREAD

**Discussion by ladeak** [dotnet/aspnetcore#57930] [General] [answered]
Is there a halt on community PRs due to the coming release? Usually there are a lot more discussions on-going PRs, but the past weeks are noticeably quiet. Have I missed some communication? When would you expect things to resume normal?

**Comment by martincostello** [comment]
Not that I'm aware of, but the team will be busy so might not have as much time to look at them.

Any community PRs that are merged though will be for .NET 10 at this point as .NET 9 branched last month ahead of RC1.

**Comment by adityamandaleeka** [comment, marked as answer]
Yea, there's no "halt" but the team has been very busy with landing .NET 9/Aspire 9 work which has reduced our capacity for reviewing incoming PRs.
